cluster/gce/util.sh

```diff
@@ -695,16 +677,16 @@ function kube-up {
   write-node-env
   create-node-instance-template
 
-  gcloud preview managed-instance-groups --zone "${ZONE}" \
+  gcloud beta compute instance-groups managed \
```

Comment:
When I tested a similar command yesterday this was working for me without the 'beta' cmd group, so I think we can remove that from the script.